### PR TITLE
[GLOSSARY] difficulty target

### PR DESCRIPTION
**Pull request by MaxHillebrand**
adds difficulty target to glossary

### CODE REVIEW

**Comment by RiccardoMasutti** [comment]
NACK.
I think Bitcoin basics should not be in Wasabi glossary.

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ Online computers are vulnerable to hackers and should not be used to store a sig
 Read more: [Cold Wasabi Protocol](/using-wasabi/ColdWasabi.md)
 :::
 
+:::details
+### Difficulty Target
+
+A Bitcoin block is only valid if its hash is below a certain difficulty target.
+Every 2016 blocks, the difficulty target is adjusted upwards [if hashing power has decreased] or downwards [if hashing power has increased].
+This ensures that regardless how much hashing power is invested into mining blocks, on average a new block is found every 10 minutes.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
This ensures that regardless of how much hashing power is invested into mining blocks, on average a new block is found every 10 minutes.
```

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ Online computers are vulnerable to hackers and should not be used to store a sig
 Read more: [Cold Wasabi Protocol](/using-wasabi/ColdWasabi.md)
 :::
 
+:::details
+### Difficulty Target
+
+A Bitcoin block is only valid if its hash is below a certain difficulty target.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
One of the requirements for a Bitcoin block to be valid is its hash should be below a certain difficulty target.
```

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
One of the requirements for a Bitcoin block to be valid is its hash must be below a certain difficulty target.
```

good, I change `should` to `must`.

**Comment by MaxHillebrand** [comment]
`Difficulty Target` is not mentioned in the docs, other than in the glossary, so I think it is not important to add.

**Comment by yahiheb** [comment]
I think it is worth mentioning it in the `bitcoin in general` section.

**Comment by nopara73** [comment]
Deleted mine.

**Pull request closed without merging**